magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/Api.scala

```diff
@@ -72,6 +73,7 @@ class Api(val webHookActorOption: Option[ActorRef], val authClient: AuthApiClien
   val routes = handleExceptions(myExceptionHandler) {
       pathPrefix("v0") {
         path("ping") { complete("OK") } ~
+          pathPrefix("tenants") { new TenantsServiceRO(config, system, materializer).route } ~
```

Comment:
Shouldn't this go above, near the other `RO` services?

Reply:
yes.

Reply:
fixed